front/app/types/ExportToCsvMutation.ts

```diff
@@ -0,0 +1,25 @@
+/* tslint:disable */
+// This file was automatically generated and should not be edited.
```

Comment:
Seems crazy to me how many of these types don't get manually committed along with the commit. Another thing that good CI would prevent getting merged.